### PR TITLE
Default pr list to open prs

### PR DESCRIPTION
**Pull request by jpribyl**
### Describe what this PR does / why we need it
Problem: In a production environment, it's common to have 1000s of PRs and unless I use a browser, I'm not going to know what is currently open and what is not. It's tedious to always type `Octo pr list states=OPEN`

Solution: default the state argument to "OPEN" - this matches the behavior of the gh-cli's `gh pr list` https://cli.github.com/manual/gh_pr_list

### Does this pull request fix one issue?

NONE

### Describe how you did it
Added a default for `opts.states` in `menu.pull_requests`

### Describe how to verify it

1. Open vim
2. Run `Octo pr list` to verify that open works
3. Run `Octo pr list states=CLOSED` to verify that closed works
4. Run `Octo pr list states=MERGED` to verify that merged works

### CODE REVIEW

**Review by pwntester — CHANGES_REQUESTED** [review]
Thanks! It makes sense but can you please add a similar default for issues?

File: lua/octo/menu.lua [hunk 1]
```diff
@@ -58,6 +58,8 @@ local function get_filter(opts, kind)
       val = json.stringify(val)
       val = string.gsub(val, '"OPEN"', "OPEN")
       val = string.gsub(val, '"CLOSED"', "CLOSED")
+      val = string.gsub(val, '"MERGED"', "MERGED")
+      val = string.gsub(val, '"ALL"', "ALL")
```
Comment (pwntester) [inline lua/octo/menu.lua]: test

comment at commit level (line 62) that will be deleted in commit afterwards

File: lua/octo/menu.lua [hunk 2]
```diff
@@ -58,6 +58,8 @@ local function get_filter(opts, kind)
       val = json.stringify(val)
       val = string.gsub(val, '"OPEN"', "OPEN")
       val = string.gsub(val, '"CLOSED"', "CLOSED")
+      val = string.gsub(val, '"MERGED"', "MERGED")
```
Comment (pwntester) [inline lua/octo/menu.lua:61]: this comment should not be deleted

File: lua/octo/menu.lua (hunk 1 above)
Comment (pwntester) [inline lua/octo/menu.lua]: test

File: lua/octo/menu.lua (hunk 2 above)
Comment (pwntester) [inline lua/octo/menu.lua:61]: test1

Comment (pwntester) [inline lua/octo/menu.lua:61]: test1

File: lua/octo/menu.lua (hunk 1 above)
Comment (pwntester) [inline lua/octo/menu.lua]: test2

Comment (pwntester) [inline lua/octo/menu.lua]: test2

Comment (pwntester) [inline lua/octo/menu.lua]: 'test1'

Comment (pwntester) [inline lua/octo/menu.lua]: test1
test2

File: lua/octo/menu.lua [hunk 3]
```diff
@@ -58,6 +58,7 @@ local function get_filter(opts, kind)
...
```
Comment (pwntester) [inline lua/octo/menu.lua:61]: test

File: lua/octo/menu.lua (hunk 2 above)
Comment (pwntester) [inline lua/octo/menu.lua:61]: test1
test2

File: lua/octo/menu.lua (hunk 3 above)
Comment (pwntester) [inline lua/octo/menu.lua:61]: test1
test2

File: lua/octo/menu.lua (hunk 1 above)
Comment (pwntester) [inline lua/octo/menu.lua]: test2

File: lua/octo/menu.lua (hunk 2 above)
Comment (pwntester) [inline lua/octo/menu.lua:61]: test2

File: lua/octo/menu.lua [hunk 4]
```diff
@@ -58,6 +58,8 @@ local function get_filter(opts, kind)
       val = json.stringify(val)
       val = string.gsub(val, '"OPEN"', "OPEN")
       val = string.gsub(val, '"CLOSED"', "CLOSED")
```
Comment (pwntester) [inline lua/octo/menu.lua:60]: test2

**Comment by pwntester** [comment]
Thanks!

**Comment by jpribyl** [comment]
@pwntester oops, I didn't quite get around to adding a similar default for issues! I will open another PR

**Comment by jpribyl** [comment]
nvm, looks like you did it :+1: thanks for taking care of that!

**Comment by pwntester** [comment]
No problem, thanks for the PR!

**Pull request merged**